CMakeLists.txt

```diff
@@ -136,11 +144,11 @@ if (NMEA_BUILD_STATIC_LIB)
         target_compile_definitions(nmea PRIVATE -DENABLE_${PARSER})
         string(TOLOWER ${PARSER} PARSER)
         add_custom_command(TARGET nmea PRE_LINK
-            COMMAND ${CMAKE_OBJCOPY} --redefine-sym init=nmea_${PARSER}_init ${PARSER}.c.o
-            COMMAND ${CMAKE_OBJCOPY} --redefine-sym parse=nmea_${PARSER}_parse ${PARSER}.c.o
-            COMMAND ${CMAKE_OBJCOPY} --redefine-sym set_default=nmea_${PARSER}_set_default ${PARSER}.c.o
-            COMMAND ${CMAKE_OBJCOPY} --redefine-sym allocate_data=nmea_${PARSER}_allocate_data ${PARSER}.c.o
-            COMMAND ${CMAKE_OBJCOPY} --redefine-sym free_data=nmea_${PARSER}_free_data ${PARSER}.c.o
+            COMMAND ${CMAKE_OBJCOPY} --redefine-sym init=nmea_${PARSER}_init ${PARSER}.c.obj
+            COMMAND ${CMAKE_OBJCOPY} --redefine-sym parse=nmea_${PARSER}_parse ${PARSER}.c.obj
+            COMMAND ${CMAKE_OBJCOPY} --redefine-sym set_default=nmea_${PARSER}_set_default ${PARSER}.c.obj
+            COMMAND ${CMAKE_OBJCOPY} --redefine-sym allocate_data=nmea_${PARSER}_allocate_data ${PARSER}.c.obj
+            COMMAND ${CMAKE_OBJCOPY} --redefine-sym free_data=nmea_${PARSER}_free_data ${PARSER}.c.obj
```

Comment:
Changing .o to .obj fixes the build on Windows but breaks it on Linux and macOS. Suggest using `${CMAKE_C_OUTPUT_EXTENSION}` instead of hardcoding particular extension.

Reply:
Fixed in a4f0c0df08655be0bedbee8426281ac8558de057